packages/devtools-launchpad/src/utils/plural-form.js

```diff
@@ -0,0 +1,193 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'm sorry, I think I misunderstood the purpose of `PluralForm`. My assumption was that it would be used by L10N and not exposed directly. In this case, we should move it back to `devtools-modules` or `devtools-sham-modules`